content/_index/coffeeeditorfeatures/diagrameditor.md

```diff
@@ -0,0 +1,9 @@
++++
+title = "Diagram editor"
+weight = 30
+
+[asset]
+  icon = "fas fa-project-diagram"
++++
+
+The coffee editor provides an graphical diagram editor. The example editor allows specifying the behavior of a coffee maching using a flow chart like notation. The diagram editor is based on [the graphical language server platform (Eclipse GLSP)](https://www.eclipse.org/glsp/). Double click the file "superbrewer3000.coffeenotation" in the coffee editor to try out the diagram editor!
```

Comment:
```suggestion
The coffee editor provides a graphical diagram editor. The example editor allows specifying the behavior of a coffee maching using a flow chart like notation. The diagram editor is based on [the graphical language server platform (Eclipse GLSP)](https://www.eclipse.org/glsp/). Double click the file "superbrewer3000.coffeenotation" in the coffee editor to try out the diagram editor!
```

Reply:
You can also open the graphical editor from the tree editor using a right click menu on the workflow node.